IBAnimatable/SystemCameraIrisAnimator.swift

```diff
@@ -26,12 +23,15 @@ public class SystemCameraIrisAnimator: NSObject, AnimatedTransitioning {
     case .Open:
       self.transitionAnimationType = .SystemCameraIris(hollowState: .Open)
```

Comment:
@tbaranes yes, I will change it to make it more consistent, thanks for picking up this.